gstreamer-sdp/src/m_i_k_e_y_payload_s_p_param.rs

```diff
@@ -12,7 +12,7 @@ use ffi;
 pub struct MIKEYPayloadSPParam(ffi::GstMIKEYPayloadSPParam);
 
 impl MIKEYPayloadSPParam {
-    pub fn new(type_: u8, len: u8, val: u8) -> MIKEYPayloadSPParam {
+    pub fn new(type_: u8, len: u8, mut val: u8) -> MIKEYPayloadSPParam {
```

Comment:
This is not a simple `u8` but a `&[u8]`. I don't think we need a constructor at all for this as `gst_mikey_payload_sp_add_param()` internally creates it anyway.